docker/Dockerfile.clightning

```diff
@@ -70,8 +70,7 @@ RUN mkdir lnprototest
 COPY . lnprototest
 
 RUN cd lnprototest && \
-    poetry config virtualenvs.create false && \
     poetry install
 RUN cd lnprototest && ls -lha
 
-CMD ["./lnprototest/docker/entrypoint.sh"]
+CMD ["poetry", "run", "./docker/entrypoint.sh"]
```

Comment:
Wrong place! You need to change the `entrypoint.sh` and use `poetry run` only in the command that needs it